model/annotations.go

```diff
@@ -91,7 +91,7 @@ type PdfAnnotationText struct {
 // (Section 12.5.6.5 p. 403).
 type PdfAnnotationLink struct {
 	*PdfAnnotation
-	A          core.PdfObject
+	A          *PdfAction
```

Comment:
This seems like could be a breaking change, typically we use the core.PdfObject only until we need to get the model (lazy way).  This also reduces overhead required in deriving the model (which might not be used).  Also this can be more error prone.

For example, `PdfPage` has `Annots` of type `core.PdfObject` and a method `GetAnnotations` that loads the annotations from `Annots` and then caches into `page.annotations` (so only need to be done once).

Does that make sense?  In this case could have an unexported `action *PdfAction` and a method such as `GetAction` (unless you can think of a more fitting way).

This would be the best approach for this, and also backwards compatible.